RubberduckTests/Refactoring/MoveCloserToUsageTests.cs

```diff
@@ -767,7 +767,6 @@ public void MoveCloserToUsageRefactoring_TargetInDifferentNonStandardModule()
         [Test]
         [Category("Refactorings")]
         [Category("Move Closer")]
-        [Ignore("For some reason the reference is not recognized by the resolver in this test.")]
```

Comment:
🎉 Nice to see less ignored tests

Reply:
I will go through the other ignored tests and see whether more of them work now.

Reply:
This has already led to fixing issue #3937.

Reply:
I ahev reviewed the remaining ignored test and could unignore a few, mostly fixed in previous PRs.